backend/src/test/java/com/ody/route/service/RouteClientCircuitBreakerTest.java

```diff
@@ -0,0 +1,68 @@
+package com.ody.route.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import com.ody.common.BaseRedisTest;
+import com.ody.route.domain.RouteClientKey;
+import java.util.concurrent.TimeUnit;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+class RouteClientCircuitBreakerTest extends BaseRedisTest {
+
+    @Autowired
+    private RouteClientCircuitBreaker circuitBreaker;
+
+    @MockBean
+    @Qualifier("odsay")
+    private RouteClient routeClient;
+
+    @DisplayName("실패 횟수를 기록하고 TTL을 31분으로 설정한다.")
+    @Test
+    void recordFailCountInMinutes() {
+        circuitBreaker.recordFailCountInMinutes(routeClient);
+
+        String failClientKey = RouteClientKey.getFailKey(routeClient);
+        int failureCount = redisTemplate.getKeyCount(failClientKey);
+        Long ttlMinutes = redisTemplate.getExpire(failClientKey, TimeUnit.MINUTES);
+
+        // 지연 시간 때문에 TTL을 범위로 테스트
+        assertAll(
+                () -> assertThat(failureCount).isEqualTo(1),
+                () -> assertThat(ttlMinutes).isGreaterThanOrEqualTo(30).isLessThanOrEqualTo(31)
```

Comment:
뭔가 있을 것 같아서 찾아보았는데 isBetween도 있더라고요!

```suggestion
                () ->  assertThat(ttlMinutes).isBetween(30L, 31L)

```

Reply:
오호 isBetween() 확인!